compoments/report-repair/repair-availability.js

```diff
@@ -12,37 +12,29 @@ import { serviceName } from '../../helpers/constants';
 import ErrorSummary from '../errorSummary';
 import {customerServicesTelephoneNumber} from '../../globals'
 import ComponentHeader from '../componentHeader';
-import { getRepairType } from '../../helpers/repairType';
 
-const RepairAvailability = ({ handleChange, values, fromDate }) => {
+const RepairAvailability = ({ handleChange, values ,fromDate, stepName, submitChangeAppointment, isSubmit=false}) => {
```

Comment:
The prop `stepName` should be just name as this component doesn't understand the notion of a step.